microcosm_flask/conventions/landing.py

```diff
@@ -0,0 +1,22 @@
+"""
+Landing Page convention.
+
+"""
+from microcosm_flask.templates.landing import template
+from jinja2 import Template
+
+
+def configure_landing(graph):
+
+    @graph.flask.route("/")
+    def render_landing_page():
+        """
+        Render landing page
+
+        """
+        return Template(template).render(
+            service_name=graph.metadata.name,
+            swagger_version=graph.config.swagger_convention.version,
+            description=graph.config.get('description', None),
```

Comment:
This is a good start, but I'd love to make this not require configuration.

One possible direction:
 - See if we can programmatically inspect any of the data set in `setup.py` (e.g. `long_description`)
 - Write something to populate that field in `setup.py` from a `README.md` (and move developer information into a different file, e.g `CONTRIBUTING.md` or `DEVELOPING.md`)